.github/workflows/test.yaml

```diff
@@ -9,14 +8,24 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - setup: "3.9"
+            tox: "py39"
+          - setup: "3.10"
+            tox: "py310"
+          - setup: "3.11"
+            tox: "py311"
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Python 3.9
-        uses: actions/setup-python@v4
-        with:
-          python-version: 3.9
       - name: Install dependencies
         run: |
-          python -m pip install -r setup_requirements.txt
+          python -m pip install --upgrade pip
+          python -m pip install tox build
       - name: Run unit tests
-        run: tox -e py
\ No newline at end of file
+        run: tox -e py
+      # TODO: switch to tox
+      - name: Build wheel
```

Comment:
Just wondering what is the motivation to have a build wheel section for the test workflow. Isnt this workfow just for unit tests? If the tests pass, but the wheel building fail, it will report failure as a whole, or just that the wheel building has failed (with the appropriate error)>?

Reply:
My mistake. thanks!
Build wheel was part of the build image.